configure.ac

```diff
@@ -6686,11 +6686,10 @@ int main(void)
 ]])],
 [ac_cv_strftime_c99_support=yes],
 [ac_cv_strftime_c99_support=no],
-[ac_cv_strftime_c99_support=no])])
-if test "$ac_cv_strftime_c99_support" = yes
+[ac_cv_strftime_c99_support=yes])])
+if test "$ac_cv_strftime_c99_support" = no
 then
-  AC_DEFINE([Py_STRFTIME_C99_SUPPORT], [1],
-  [Define if C99-specific strftime specifiers are supported.])
+  AC_MSG_ERROR([Python requires C99-compatible strftime specifiers])
 fi
```

Comment:
I tried a couple different approaches to avoid `ac_cv_strftime_c99_support` entirely or avoid using `=yes` when cross-compiling but don't really know how to use autoconf yet. If a different approach is desired, lmk.

Reply:
This seems fine to me; if the check fails, we can't build.

However, we _should_ also be fine with just removing this check. As you already pointed out, we require C99 support.

(I'm not sure what I meant about the `libm` reference in the other PR 😆)

Reply:
Agree — I don't think it's a big deal either way. It seems easy to remove if it's causing a problem.